### PR TITLE
Styles added for image in ProductPage

### PR DESCRIPTION
**Pull request by fromITzerotoIThero**
Changed styles for the image, would you take a look and tell me it's a good and robust way of changing image size?

### CODE REVIEW

File: src/pages/ProductPage/ProductPage.module.css [hunk 1]
```diff
@@ -34,6 +34,15 @@
     font-size: 0.75rem;
 }
 
+.img_container {
+    overflow: hidden;
```
Comment (piotrwicijowski) [inline src/pages/ProductPage/ProductPage.module.css:38]: Is the overflow necessary?

File: src/pages/ProductPage/ProductPage.module.css [hunk 2]
```diff
@@ -34,6 +34,15 @@
...
+}
+
+.img_container img {
+    width: 100%;
+    height: auto;
```
Comment (piotrwicijowski) [inline src/pages/ProductPage/ProductPage.module.css:43]: Is the height:auto necessary?

**Review by piotrwicijowski — COMMENTED** [review]
Based on my quick tests just setting the width to 100% should be sufficient

File: src/pages/ProductPage/ProductPage.module.css (hunk 1 above)
Comment (fromITzerotoIThero) [reply src/pages/ProductPage/ProductPage.module.css:38]: Checked - nope, it's not! :)

File: src/pages/ProductPage/ProductPage.module.css (hunk 2 above)
Comment (fromITzerotoIThero) [reply src/pages/ProductPage/ProductPage.module.css:43]: Same with height - I'm deleting it :)

**Comment by piotrwicijowski** [comment]
Hi G.
First of all, you need to turn this draft PR into a regular PR. You can do that by clicking on "Ready for review". Once this is done, I can approve that. After that you will have the option to "merge pull request" and "delete the branch" - all from this pull request view in github.

**Pull request merged**